packages/shared/src/logic/utils.ts

```diff
@@ -0,0 +1,106 @@
+import { differenceInDays, endOfToday, format } from 'date-fns';
+import emojiRegex from 'emoji-regex';
+import _ from 'lodash';
+
+type App = 'chat' | 'heap' | 'diary';
+
+export function checkNest(nest: string) {
```

Comment:
Can these be removed or moved to urbit package? Trying to keep all the urbit utils colocated.

Reply:
are you referring to checkNest and nestToFlag in particular?